composer.json

```diff
@@ -22,10 +22,8 @@
   "license": "MIT",
   "autoload": {
     "psr-4": {
-      "Opulence\\": [
-        "src/Opulence",
-        "tests/src/Opulence"
-      ]
+      "Opulence\\": "src/Opulence",
+      "Opulence\\Tests\\": "tests/src/Opulence"
```

Comment:
Probably this should go to `autoload-dev` so composer optimize don't need test class.